Assets/Scripts/Framework/Buttons/PointsSystem/AddPointsButton.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+namespace ProefExamen.Framework.Buttons.PointsSystem
+{
+    /// <summary>
+    /// A class that adds points to the points system when the button is pressed for testing purposes.
+    /// </summary>
+    public class AddPointsButton : BasicButton
+    {
+        [SerializeField]
+        private int _pointsToAdd = 1;
+
+        /// <summary>
+        /// <inheritdoc/>
```

Comment:
Is dit correct? Wat houd `<inheritdoc/>` in?

Reply:
It inherits the summary from the base class.

Reply:
👍